sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/QueryPlanRetriever.java

```diff
@@ -51,7 +51,8 @@ class QueryPlanRetriever {
                                                                QueryFeature.Top.name() + ", " +
                                                                QueryFeature.DCount.name() + ", " +
                                                                QueryFeature.NonValueAggregate.name() + ", " +
-                                                               QueryFeature.NonStreamingOrderBy.name();
+                                                               QueryFeature.NonStreamingOrderBy.name() + ", " +
+                                                               QueryFeature.HybridSearch.name();
```

Comment:
do we also need to add `CountIf` query feature here?

Reply:
Added it, it was in my local changes, forgot to push it.